salt/modules/aptpkg.py

```diff
@@ -2867,6 +2867,16 @@ def info_installed(*names, **kwargs):
     failhard = kwargs.pop('failhard', True)
     kwargs.pop('errors', None)                # Only for compatibility with RPM
     attr = kwargs.pop('attr', None)           # Package attributes to return
+
+    # status is needed to see if a package is installed. So we have to add it,
+    # even if it's excluded via attr parameter. Otherwise all packages are
+    # returned.
+    if attr:
+        attr_list = attr.split(',')
```

Comment:
[nitpick] If the order of the attributes is not relevant, this could be a Set instead of a List (removes the next `if` statement)

Reply:
Good point. Not sure if it's really faster though. But let's change it.